napari/_vispy/experimental/_tests/test_vispy_tiled_image.py

```diff
@@ -47,7 +47,7 @@ def test_tiled_screenshot(qtbot, monkeypatch, make_napari_viewer, dtype):
         data, multiscale=True, contrast_limits=[0, 200], colormap='blue'
     )
 
-    visual = viewer.window._qt_viewer.layer_to_visual[layer]
+    visual = viewer.window._qt_viewer._layer_to_visual[layer]
```

Comment:
`canvas.layer_to_visual`?

It looks wrong for me and this part of code is untested.

Reply:
Adjusted with [a45c751](https://github.com/napari/napari/pull/5432/commits/a45c751d844c8bf6447c563843e7e7f2353eabb0). Was a remnant from before the discussion of having this as private or not. Still worked due to backward compatibility.